includes/class-taxjar-customer-record.php

```diff
@@ -261,6 +233,47 @@ public function get_data_from_object() {
 		return $customer_data;
 	}
 
+	/**
+	 * Retrieves the name of the customer.
+	 * Falls back to username if no shipping, billing or account names are available.
+	 *
+	 * @return string - Customer's name
+	 */
+	public function get_customer_name() {
```

Comment:
This method contains the same logic as was pulled out of the get_data_from_object method. The only exception is that a fallback to the username was added.